src/Transition/CreateNewIDFUsingRulesV9_6_0.f90

```diff
@@ -412,6 +420,14 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with R, insert the rules here
 
               ! If your original object starts with S, insert the rules here
+              
+              CASE('SIZING:SYSTEM')
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                nodiff=.false.
+                OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                IF (MakeUPPERCase(OutArgs(27)) == "VENTILATIONRATEPROCEDURE") THEN
+                  OutArgs(27) = "Standard62.1VentilationRateProcedure"
+                ENDIF
```

Comment:
Transition rule for changing the `Sizing:System`'s `System Outdoor Air Method` from  `VentilationRateProcedure` to `Standard62.1VentilationRateProcedure`